docs/manual/advanced-features.tex

```diff
@@ -285,6 +285,23 @@
 Section~\ref{climb-to-top}.
 
 
+\subsectionAndLabel{Default qualifier on constructor declarations}{constructor-defaults}
+
+If a constructor declaration is unannotated, it defaults to the same type as that of
+its enclosing class (rather than the default qualifier in hierarchy).
+For example: the Tainting type checker (Chapter~\ref{tainting-checker}) has \code{@Tainted}
```

Comment:
For consistency with that chapter, change `Tainting type checker` to `Tainting Checker`.